examples/sdk-examples/src/examples/attributeFilter/index.tsx

```diff
@@ -134,10 +151,19 @@ export const AttributeFilter = (): JSX.Element => (
             between filters.
         </p>
 
-        <p>Dependency implemented via Placeholders.</p>
+        <p>
+            Dependency between parent filter and child filter is implemented via Placeholders and there is
+            preselect value for parent filter demonstrated in this example.
+        </p>
+
+        <p>
+            <strong>
+                ParentFilter property MUST be a filter defined by uris because of backend limitations.
```

Comment:
no problem with the limitation (for now) however i suggest to reword this; just say something like "current limitation is that the parent filter must be specify attribute elements using their URI"